src/theory/bv/kinds

```diff
@@ -43,6 +43,8 @@ well-founded BITVECTOR_TYPE \
 ## conversion to bit-vector from vector of Booleans kind
 operator BITVECTOR_BB_TERM 1: "create bit-vector from vector of Booleans"
 
+## size
+operator BITVECTOR_SIZE 1 "Size of bitvector"
```

Comment:
You should specify the type rule for BITVECTOR_SIZE in this file as well.

Reply:
What kind of rules do you have in mind? I looked through other operators in that file and they don't seem to specify a type rule.

Reply:
See for example: https://github.com/cvc5/cvc5/blob/main/src/theory/bv/kinds#L178

Reply:
done